src/raster/r.forcircular/r.forcircular

```diff
@@ -0,0 +1,2039 @@
+#!/usr/bin/env python3
```

Comment:
Please add the missing `.py` file extension to `r.forcircular`

Reply:
Done